File: app/src/main/java/fr/husta/android/dockersearch/AppConstants.java
package fr.husta.android.dockersearch;


public interface AppConstants
{

    /**
     * https://developer.chrome.com/multidevice/android/customtabs
     */
    boolean USE_CHROME_CUSTOM_TABS = true;

    /**
     * Default page size for {@link fr.husta.android.dockersearch.docker.DockerSearchRestService#searchImages(java.lang.String, int)}.
     * <br/>
     * MAX = 100.
     */
    int IMAGE_SEARCH_PAGE_SIZE = 30;

    int TAG_LIST_PAGE_SIZE = 50;

}

File: app/src/main/java/fr/husta/android/dockersearch/AppConstants.kt
package fr.husta.android.dockersearch

object AppConstants {

    /**
     * https://developer.chrome.com/multidevice/android/customtabs
     */
    const val USE_CHROME_CUSTOM_TABS = true

    /**
     * Default page size for [fr.husta.android.dockersearch.docker.DockerSearchRestService.searchImages].
     * <br></br>
     * MAX = 100.
     */
    const val IMAGE_SEARCH_PAGE_SIZE = 30

    const val TAG_LIST_PAGE_SIZE = 50

}
